static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -85,17 +105,136 @@ export class AccessMethod extends Component {
 
     let selectedHarmonyMethodName = ''
     if (selectedAccessMethod
-        && isHarmony
-        && accessMethods[selectedAccessMethod].name) {
+      && isHarmony
+      && accessMethods[selectedAccessMethod].name) {
       selectedHarmonyMethodName = accessMethods[selectedAccessMethod].name
     }
 
+    const granuleExtent = false
+    const sampleGrid = 'UTM'
+    const rasterResolution = 90
+    // We only want to allow for the processing of the first 10 granules of a collection for Swodlr
+    const utmRasterOptions = [
+      {
+        title: '90 Meters',
+        value: 90
+      },
+      {
+        title: '120 Meters',
+        value: 120
+      },
+      {
+        title: '125 Meters',
+        value: 125
+      },
+      {
+        title: '200 Meters',
+        value: 200
+      },
+      {
+        title: '250 Meters',
+        value: 250
+      },
+      {
+        title: '500 Meters',
+        value: 500
+      },
+      {
+        title: '1000 Meters',
+        value: 1000
+      },
+      {
+        title: '2500 Meters',
+        value: 2500
+      },
+      {
+        title: '5000 Meters',
+        value: 5000
+      },
+      {
+        title: '10000 Meters',
+        value: 10000
+      }
+    ]
+
+    const geoRasterOptions = [
+      {
+        title: '3 arc-seconds',
+        value: 3
+      },
+      {
+        title: '4 arc-seconds',
+        value: 4
+      },
+      {
+        title: '5 arc-seconds',
+        value: 5
+      },
+      {
+        title: '6 arc-seconds',
+        value: 6
+      },
+      {
+        title: '8 arc-seconds',
+        value: 8
+      },
+      {
+        title: '15 arc-seconds',
+        value: 15
+      },
+      {
+        title: '30 arc-seconds',
+        value: 30
+      },
+      {
+        title: '60 arc-seconds',
+        value: 60
+      },
+      {
+        title: '180 arc-seconds',
+        value: 180
+      },
+      {
+        title: '300 arc-seconds',
+        value: 300
+      }
+    ]
+
+    const {
+      granules: projectCollectionGranules = {}
+    } = projectCollection
+
+    const {
+      addedGranuleIds = [],
+      allIds: granulesAllIds = []
+    } = projectCollectionGranules
+
+    let granulesToDisplay = []
+
+    if (addedGranuleIds.length > 0) {
+      granulesToDisplay = addedGranuleIds
+    } else {
+      granulesToDisplay = granulesAllIds.slice(0, 10)
+    }
+
+    const granuleList = []
```

Comment:
I think this is fine right now. We may wanna just rename this var to be more swodlr specific since that's all its being used for but, I think this file AccessMethod is probably the highest priority in general just something we gotta refactor and split apart; unrelated to this immediate work item